server/prisma/schema.prisma

```diff
@@ -11,6 +11,15 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model instance_settings {
+  id                    Int    @id
```

Comment:
```suggestion
  id                    Int     @id @default(autoincrement())
```
Only thing I can think of, without trying to go into how it would be used.

Reply:
Wait, I was planning to have one instance, but we can keep the change if we have save state for multiple instance setting 🤣